master/buildbot/steps/shellsequence.py

```diff
@@ -31,15 +31,23 @@
 class ShellArg(results.ResultComputingConfigMixin):
     publicAttributes = (
         results.ResultComputingConfigMixin.resultConfig +
-        ["command", "logfile"])
+        ["command", "logname"])
 
-    def __init__(self, command=None, logfile=None, **kwargs):
+    def __init__(self, command=None, logname=None, logfile=None, **kwargs):
         name = self.__class__.__name__
         if command is None:
             config.error("the 'command' parameter of %s "
                          "must not be None" % (name,))
         self.command = command
-        self.logfile = logfile
+
+        self.logname = logname
+        if logfile is not None:
+            config.warnDeprecated('1.0.1', "logfile is deprecated, use logname")
+            if self.logname is not None:
```

Comment:
I can see those lines are uncovered. We must add some unit tests to make sure that logfile is still working (i.e no mistake error in the warning)

Reply:
You can see here example how to flush and verify the warnings:
https://github.com/buildbot/buildbot/blob/master/master/buildbot/test/unit/test_worker_transition.py#L107